product-configuration-extension/locales/en.default.json.liquid

```diff
@@ -0,0 +1,3 @@
+{
+  "welcome": "Welcome to the {% raw %}{{extensionPoint}}{% endraw %} extension!"
```

Comment:
As part of the Unified Config project we now need to stop saying extension points but extension targets
```suggestion
  "welcome": "Welcome to the {% raw %}{{target}}{% endraw %} extension!"
```